h2o-algos/src/main/java/hex/ensemble/StackedEnsemble.java

```diff
@@ -97,7 +97,7 @@ private Frame prepareLevelOneFrame(String levelOneKey, Model[] baseModels, Frame
           Log.warn("Failed to find base model " + baseModel + " predictions; skipping: " + baseModelPreds._key);
           continue;
         }
-
+        // TODO: Add fold_column here
```

Comment:
Isn't fold column already added?

Reply:
Yep, I can delete that comment (it's out of date now).  Thx.